execution/execution.go

```diff
@@ -66,6 +103,20 @@ type NamedExpression interface {
 	Name() octosql.VariableName
 }
 
+type DummyExpression struct {
```

Comment:
Is this not really a constant expression?

Reply:
By asking this question do you mean we already have such a thing defined, or do you mean that the name should be different?

Reply:
Name should be different.